src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneOptionsView.js

```diff
@@ -0,0 +1,118 @@
+import {
+  loc,
+  createButton } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'phone-authenticator-challenge',
```

Comment:
`phone-authenticator-challenge-data`
or 
`authenticator-challenge-data-phone`

Reply:
Thought of keeping `phone-authenticator-challenge` `className` for both selection view (data view) and enter code views. Not sure what value it may add by keeping it separate, the common factor is phone and i don't see css growing too separate for both of these.. any thoughts?

Reply:
sure. not big deal. I just thought might be better to match css with page name.